Samsung/A32-SystemUI/res/values/bools.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_enableWifiDisplay">true</bool>
+    <bool name="config_adaptive_display_solution_enabled">true</bool>
+    <bool name="config_all_around_sensing_enabled">false</bool>
+    <bool name="config_big_data_logging_enabled">false</bool>
+    <bool name="config_dynamic_automatic_brightness_available">true</bool>
```

Comment:
that's not in systemui

Reply:
okay i'll remove those. sorry i am not an expert in overlays

Reply:
configs removed